integration/helpers/app_instance_table.go

```diff
@@ -58,8 +59,8 @@ func ParseV3AppProcessTable(input []byte) AppTable {
 			// instance row
 			columns := splitColumns(row)
 			details := ""
-			if len(columns) >= 7 {
-				details = columns[6]
+			if len(columns) >= 8 {
```

Comment:
Is it 8 the number of columns we have? if that's the case, could we create a constant for that instead of hardcoding it ?